pkg/storage/sds/models/models.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package models
```

Comment:
these structs can be promoted upward one level, into the `sds` package

Reply:
Done!